app/javascript/comments/Comments.vue

```diff
@@ -50,19 +50,11 @@ const googleLoginOrigin = [
 </script>
 
 <style scoped>
-.google-login-form {
-  display: inline;
-}
```

Comment:
I was initially going to do the test replacing this class, but it seems not to be needed (anymore?).